src/app/checkout/Checkout.tsx

```diff
@@ -168,18 +177,24 @@ class Checkout extends Component<CheckoutProps & WithCheckoutProps & WithLanguag
 
     render(): ReactNode {
         const { error } = this.state;
+        let errorModal = null;
+
+        if (error) {
```

Comment:
if you undo the changs in `src/app/common/error/ErrorModal.tsx `, you don't need any of this new code. You are setting the error title in https://github.com/bigcommerce/checkout-js/pull/429/files#diff-0f46a26e5e64c712f71cf19d3044b4b4d573f962b90afb03efca90c9c607fdd3R145 and the modal already has logic to render the error title, no need to pass it.